test.js

```diff
@@ -160,17 +160,17 @@ tests.forEach(function (test) {
   })
 
   test('TXT record', function (dns, t) {
-    var data = Buffer.from('black box')
+    var data = [Buffer.from('black box')]
```

Comment:
Mind adding a test here where the array contains more than one item also?

Reply:
No worries.